lib/tasks/lint.rake

```diff
@@ -1,5 +1,5 @@
 desc "Run govuk-lint on all files"
 task "lint" do
-  sh "govuk-lint-ruby --format clang"
+  sh "govuk-lint-ruby app config lib spec --format clang"
```

Comment:
What about adding rails linting as per https://github.com/alphagov/content-publisher/blob/master/lib/tasks/lint.rake#L5

Reply:
Will make the change now.